timepiece/templates/timepiece/base.html

```diff
@@ -78,9 +78,9 @@
         </footer>
     {% endblock footer %}
 
-    <script charset="utf-8" src="{% static "timepiece/js/sticky_table_header.js" %}"></script>
     {% block extrajs %}
     {% endblock extrajs %}
+    <script charset="utf-8" src="{% static "timepiece/js/sticky_table_header.js" %}"></script>
```

Comment:
I think it is better for this to remain before the `extrajs` block. This would allow a specific template to override what the sticky headers are doing, if needed.